content/hi/docs/reference/glossary/admission-controller.md

```diff
@@ -0,0 +1,21 @@
+---
+title: एडमिशन नियंत्रक (Admission Controller)
+id: admission-controller
+date: 2019-06-28
+full_link: /docs/reference/access-authn-authz/admission-controllers/
+short_description: >
+  यह कोड का अंश जो ऑब्जेक्ट शुरू होने से पहले कुबेरनेट्स API सर्वर पर किए गए अनुरोध को संसाधित करता है।
+
+aka:
+tags:
+- extension
+- security
+---
+यह कोड का अंश जो ऑब्जेक्ट शुरू होने से पहले कुबेरनेट्स API सर्वर पर किए गए अनुरोध को संसाधित करता है।
+
+<!--more-->
+
+एडमिशन नियंत्रक कुबेरनेट्स API सर्वर के लिए  विन्यस्त करने योग्य हैं और यह "सत्यापन" (validating), "म्युटेटिंग" (mutating) या दोनों हो सकते हैं।
```

Comment:
`एडमिशन नियंत्रक कुबेरनेट्स API सर्वर के लिए  कॉन्फ़िगर करने योग्य हैं और  "सत्यापन" (validating), "म्युटेटिंग" (mutating) या दोनों हो सकते हैं।`
Some small changes which i think will be easy to understand . instead of `विन्यस्त` we can  write `कॉन्फ़िगर`.